### ISSUE TITLE
MacOS support

### ISSUE THREAD

**Issue by akatsukinoyami** [veesusmikelheir/SRML#29]
Is there any way to install on MacOS? 

Installer is `exe`-file, maybe there some way to make version, where mod would be a pack of files, that can be copied to game folder?

**Comment by YLohkuhl** [comment]
No, Mac is not supported for SRML and there has not really been any good news on people bypassing this. It is originally made for Windows.

**Comment by Atmudia** [comment]
You can try with Wine but uh
